source/uwp/Renderer/lib/AdaptiveImageRenderer.cpp

```diff
@@ -576,16 +658,23 @@ namespace AdaptiveCards::Rendering::Uwp
                             {
                                 ::AdaptiveCards::Rendering::Uwp::XamlHelpers::SetAutoImageSize(lambdaImageAsFrameworkElement,
                                                                                                lambdaParentElement,
-                                                                                               imageSourceAsBitmapSource,
+                                                                                               imageSource,
                                                                                                isVisible);
                             }
                         }
                     });
             }
             else
             {
-                ::AdaptiveCards::Rendering::Uwp::XamlHelpers::SetAutoImageSize(xamlImage, parentElement, imageSourceAsBitmapSource, isVisible);
+                ::AdaptiveCards::Rendering::Uwp::XamlHelpers::SetAutoImageSize(xamlImage, parentElement, imageSource, isVisible);
             }
         }
     }
+
+    boolean XamlBuilder::IsSvgImage(std::string url)
```

Comment:
Are there other checks we want to have here?

Reply:
this will do until we support SVG officially and add additional properties to tell us the url is to svg.

Reply:
sounds good @jwoo-msft